src/Employer/Employer.Web/Views/EditVacancyDates/EditVacancyDates.cshtml

```diff
@@ -85,7 +85,9 @@
                     <p class="govuk-body">@Model.ProgrammeName</p>
                 </div>
                 <button type="submit" class="govuk-button save-button" esfa-automation="continue-button">Preview changes</button>
-                <a asp-route="@RouteNames.VacancyEdit_Get" class="govuk-link das-button-link">Cancel</a>
+                <div class="govuk-body">
+                    <a asp-route="@RouteNames.VacancyEdit_Get" class="govuk-link das-button-link">Cancel</a>
+                </div>
```

Comment:
The `das-button-link` class should be removed, it has some internal padding so it doesn't align with the continue button. The other pages in the journey only just `govuk-link`